README.md

```diff
@@ -1,264 +1,205 @@
+  ordinal values.
+
+And the output should be a single 2D `numpy.ndarray` with the exact same shape as the `real_data`
+matrix.
+
+# Usage
+
+All you need to do in order to use the SDGym Benchmark, is to import and call the `sdgym.benchmark`
+function passing it your synthesizer function:
+
+```python
+from sdgym import benchmark
+
+scores = benchmark(my_synthesizer_function)
 ```
 
-Have below the list of included datasets and their original source:
-
-- MINIST28: Use flatten 28\*28 pixels into 784 binary columns with an extra label column.
-- MINIST12: Reshape 28\*28 pixels into 12\*12 binary columns with an extra label column.
-- Credit: Kaggle credit card fraud dataset. https://www.kaggle.com/mlg-ulb/creditcardfraud
-- Adult: Adult Dataset. https://archive.ics.uci.edu/ml/datasets/adult
-- Census: KDD Census dataset https://archive.ics.uci.edu/ml/datasets/Census-Income+(KDD)
-- News: Online News Popularity Dataset (Regression) https://archive.ics.uci.edu/ml/datasets/online+news+popularity
-- Covertype: Covertype Dataset (8 continuous + 40 binary + 1 multi) https://archive.ics.uci.edu/ml/datasets/Covertype
-- Intrusion: network intrusion detector kdd99 https://archive.ics.uci.edu/ml/datasets/kdd+cup+1999+data
-
-### Simulated data
-
-- Bivariate
-
-    - Gaussian Grid (Grid): Gaussian Mixtures arranged in a grid.
-
-    - Gaussian Grid Random (Gridr): Gaussian Mixtures arranged in a grid plus random offset.
-
-    - Gaussian Ring (Ring): Gaussian Mixtures arranged in a ring.
-
-
-    <table>
-    <tr>
-    <td>
-    <img src="https://i.imgur.com/BLe701k.jpg" width="100%">
-    </td>
-    <td>
-    <img src="https://i.imgur.com/3UkHCUp.jpg" width="100%">
-    </td>
-    <td>
-    <img src="https://i.imgur.com/rPwzsiz.jpg" width="100%">
-    </td>
-    </tr>
-    </table>
-
-- Multivariate Structured Data: Generate samples from some pre-specified common causal structures. The Bayesian networks are from [bnlearn](http://www.bnlearn.com/bnrepository/).
-    <table>
-    <tr>
-    <th>Asia</th>
-    <th>Alarm</th>
-    </tr>
-    <tr>
-    <td>
-    <figure>
-    <img src="https://i.imgur.com/iu7s4Jl.png" width="200" height="200">
-    </figure>
-    </td>
-    <td>
-    <figure>
-    <img src="https://i.imgur.com/2iPs87b.png" width="200" height="200">
-    </figure>
-    </td>
-    </tr>
-
-    <tr>
-    <th>Child</th>
-    <th>Insurance</th>
-    </tr>
-
-    <tr>
-    <td>
-    <figure>
-    <img src="https://i.imgur.com/Bj9jH9N.png" width="200" height="200">
-    </figure>
-    </td>
-    <td>
-    <figure>
-    <img src="https://i.imgur.com/a6pc8OC.png" width="200" height="200">
-    </figure>
-    </td>
-    </tr>
-    </table>
-
-## Quickstart
-
-After installing the requirements and preparing the datasets, you only need to run the following
-commands to evaluate a synthesizer:
+The output of the `benchmark` function will be a `pd.DataFrame` containing all the scores
```

Comment:
Should have a QuickStart to help people quickly run existing synthesizers on the Benchmark. Also, move QuickStart before installation for development.

Reply:
I added some explanation about the repo synthesizers and added some instructions about how to use them in the Usage section.
Regarding naming: the section is called Usage instead of Quickstart because there isn't much else in the repository right now, so it's actually both a quickstart and quickend :-)
Regarding order, this is the standard order that we are using in all the other projects: overview -> install (pip + make install + make install-develop) -> data format -> quickstart (here, usage), so I prefer to keep it in this order.